.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--  Delete any sections below that are not relevant to this PR -->
```

Comment:
Doesn't seem related! Naughty! I'll let it slip though ;-)

Reply:
[evil laugh]